dpnp/tests/test_linalg.py

```diff
@@ -2766,13 +2761,6 @@ def test_slogdet_strides(self):
         assert_allclose(sign_result, sign_expected)
         assert_allclose(logdet_result, logdet_expected)
 
-    # TODO: remove skipif when Intel MKL 2025.2 is released
-    # Skip running on CPU because dpnp uses _getrf_batch only on CPU
-    # for dpnp.linalg.det/slogdet.
-    @pytest.mark.skipif(
-        is_cpu_device() and not requires_intel_mkl_version("2025.2"),
-        reason="mkl<2025.2",
-    )
```

Comment:
seems that skipping on CPU was removed in this change

Reply:
disregard this, I understand now that the CPU skipping was introduced with MKL version skipping